.gitignore

```diff
@@ -1,2 +1,5 @@
 target
 /.bsp
+.bloop
+.metals
+metals.sbt
```

Comment:
I prefer not to add personal files to projects' `.gitignore`s.
I have a global `.gitignore` in my machine and that makes sure I never commit IDE files.